draft-ietf-tls-tls13.md

```diff
@@ -3103,6 +3135,11 @@ it in the "pre_shared_key" extension in its ClientHello
 ({{pre-shared-key-extension}}) and supplying a suitable PSK cipher
 suite.
 
+The server MAY also send a NewSessionTicket message after the handshake is
+complete, following the client CertificateVerify message. This allows the
```

Comment:
NewSessionTicket needs to follow the client's Finished.  For one, the client's CertificateVerify won't appear in every exchange.